### PR TITLE
add initial logic for hard keeper bot

### PR DESCRIPTION
**Pull request by nddeluca**
Queries chain and prints number of positions to liquidate w/ corresponding MsgLiquidates's

Running: `KAVA_KEEPER_ADDRESS=kava1ygj8karnh3dar4jxv6ef0hr473tu9y4ezeej5v KAVA_RPC_URL=https://rpc.mmm.kava.io:443 go run .`

Todo:
- loop with liquidation interval
- add message signing and sending

### CODE REVIEW

File: hard-keeper-bot/client.go [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+	resp := result.Response
+	if !resp.IsOK() {
+		return []byte{}, errors.New(resp.Log)
+	}
+
+	// TODO: why do we check length here?
+	value := result.Response.GetValue()
+	// TODO: untested logic case
+	if len(value) == 0 {
```
Comment (nddeluca) [inline hard-keeper-bot/client.go:197]: @rhuairahrighairidh I think I got this from one your clients, but can't think of a reason for it.  Unit tests don't catch this edge case yet
